src/core/qgsvectorlayerjoinbuffer.cpp

```diff
@@ -490,3 +517,137 @@ void QgsVectorLayerJoinBuffer::connectJoinedLayer( QgsVectorLayer *vl )
   connect( vl, &QgsVectorLayer::layerModified, this, &QgsVectorLayerJoinBuffer::joinedLayerModified, Qt::UniqueConnection );
   connect( vl, &QgsVectorLayer::willBeDeleted, this, &QgsVectorLayerJoinBuffer::joinedLayerWillBeDeleted, Qt::UniqueConnection );
 }
+
+bool QgsVectorLayerJoinBuffer::addFeature( const QgsFeature &feature ) const
```

Comment:
If you make QgsVectorLayerJoinBuffer a QgsFeatureSink (it already implements the required method) then you can do this method here (it's handled in QgsFestureSink)

Reply:
Do=drop

Reply:
Done